marrow/mailer/transport/ses.py

```diff
@@ -5,55 +5,55 @@
     from botocore.exceptions import ClientError
 
 except ImportError:
-    raise ImportError("You must install the boto package to deliver mail via Amazon SES.")
+    raise ImportError(
+        "You must install the boto package to deliver mail via Amazon SES.")
 
 
 __all__ = ['AmazonTransport']
 
 log = __import__('logging').getLogger(__name__)
 
 
-
-class AmazonTransport(object): # pragma: no cover
+class AmazonTransport(object):  # pragma: no cover
     __slots__ = ('ephemeral', 'config', 'region', 'connection')
-    
+
```

Comment:
"Trailing whitespace" matching the indentation level is not "trailing", and should not be stripped. This is to accomodate editors which follow the logical indentation level to implement code folding; more do this than one might think.

Reply:
Agree. Thanks for this knowledge.